README.md

```diff
@@ -83,7 +83,7 @@ Prerequisites:
 - Go >= 1.6.1
 - [godep](https://github.com/tools/godep) installed
 - libtool development headers installed
-    - Ubuntu: `apt-get install libtool-dev`
```

Comment:
Installing just `libtool` won't be sufficient, as it doesn't seem to install the headers (I think generally the `-dev` suffix on packages in ubuntu actually install the development headers).  We can change this to `libltdl-dev` though, since that's what we have in our Dockerfile for building the client.